src/hooks/Tokens.ts

```diff
@@ -1,10 +1,11 @@
 import { Currency, Token } from '@uniswap/sdk-core'
-import { CHAIN_INFO } from '@src/constants/chainInfo'
-import { L2_CHAIN_IDS, SupportedChainId, SupportedL2ChainId } from '@src/constants/chains'
-import useActiveWeb3React from 'hooks/useActiveWeb3React'
+import { useWeb3React } from '@web3-react/core'
+import { getChainInfo } from '@src/constants/chainInfo'
+import { SupportedChainId } from 'constants/chains'
 import { useCurrencyFromMap, useTokenFromMapOrNetwork } from 'lib/hooks/useCurrency'
 import { getTokenFilter } from 'lib/hooks/useTokenList/filtering'
 import { useMemo } from 'react'
+import { isL2ChainId } from '@src/utils/chains'
```

Comment:
path?

Reply:
Has to be this one, because we use different chains